### ISSUE TITLE
Suggestion: Make the large quick reply button a smaller button that spans not across the complete screen width below the last message -> Prevents creation of unwanted quick replies when dragging/dropping attachments

### ISSUE THREAD

**Issue by ghost** [thunderbird-conversations/thunderbird-conversations#1355]
* **Thunderbird version: 60.6.1 (32 Bit)**
* **Thunderbird Conversation version: 12.14.7**
* **Operating System: Windows 10**

**Current situation**
The quick reply function is a great feature. However, the quick reply button to invoke that quick reply function is very large. It is a pretty thick bar below the last message in the conversations list and spans across the entire screen.

**Nagging things about the current situation**
When using drag and drop to drag email attachments into open applications or file manager windows (for example on the Windows Task Bar or on the macOS Dock) the attachment always crosses that huge quick reply bar/button during the dragging movement. Because of that huge size there's no way of getting around that. This invokes the quick reply feature even if the user has no intention to write a reply.
It's quite annoying that every time the user has to manually cancel that unwanted quick reply and in addition to that he has to confirm the following cancel/warning message dialog box. If the user does not cancel that unwanted quick reply though, a draft is created and stored in the drafts folder and has to be manually deleted from there.

**Suggestion**
Make the large quick reply button a smaller button that spans not across the complete screen width below the last message. This new smaller quick reply button could maybe be placed next to the small normal reply button that is already there at the bottom of the last message. This could be a new setting (small/large toggle) in the preferences of this addon that gives users the choice whether they want the large or small button.

**Benefits**
Makes the described nagging things go away.

**Comment by Standard8** [comment]
I'm guessing this might be due to drag and drop. Maybe we should make it so that there's a small delay before opening e.g. waiting for hover for 400ms or something.

**Comment by Marrib** [comment]
Exactly, the d 'n' d functionality is invoked instantly. Adding a delay would probably do the trick.

**Comment by Marrib** [comment]
Maybe I may kindly ask a follow-up question: Is there a way to reduce the height of the buttons via CSS, maybe using userChrome.css?

![tbc_small](https://user-images.githubusercontent.com/40357209/63652115-09188a80-c75d-11e9-8a6a-2ca2d9a62548.jpg)
